### PR TITLE
Add section on 3D printers to FMS

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose of change
After Yet Another Suggestion To Add 3D Printers #74102 I decided to clean up my reply there and stick it in FMS.

### CODE REVIEW

**Comment by Maddremor** [comment]
Personally, my go-to argument is that you can pretty much get the same benefit as 3D-printing with woodcarving for most things.

File: doc/FREQUENTLY_MADE_SUGGESTIONS.md [hunk 1]
```diff
@@ -283,6 +283,24 @@ Storing blood for later is far outside the reach of the survivor even if they kn
...
+
+Even if you find a house or workplace with 3D printers (which is a safe assumption, because otherwise you aren't going to have a 3D printer in the first place), most of the design files you're going to find are going to be figurines, toys, and replacement parts for random appliances, not things you waThere are a number of issues with adding 3D printing. One, your assertion that ability to 3D print is based on the user's ability to operate a 3D printer assumes that they are working not just from a design, but an extremely well proven design. In reality there are many designs out there you need to heavily modify in order to get it printing, at which point you have to repeat the final stages of testing and adjusting the design in order to get a working output. This is much more the case for the survivor scenario because you're interested in working solutions instead of things that "look right".
```
Comment (Zireael07) [inline doc/FREQUENTLY_MADE_SUGGESTIONS.md:291]: > things you waThere are 

Merge error?

**Comment by Hyperseeker** [comment]
This response should've been proofread and rewritten towards a generic one before being merged. It did not receive the attention an official explanation deserves.

File: doc/FREQUENTLY_MADE_SUGGESTIONS.md (hunk 1 above)
Comment (Kamejeir) [reply doc/FREQUENTLY_MADE_SUGGESTIONS.md:291]: "nt/need" after "...just isn't that tough of a material" connects nicely to follow "wa". perhaps missed the click by a couple characters before copy-pasting his next reply

**Pull request merged**